pages.fr/common/ls.md

```diff
@@ -0,0 +1,27 @@
+# ls
+
+> Liste le contenu d'un répertoire.
+
+- Affiche un fichier par ligne:
+
+`ls -1`
+
+- Affiche tous les fichiers, ainsi que les fichiers cachés:
+
+`ls -a`
+
+- Affiche tous les fichiers avec un format détaillé (permissions, propriétaire, taille et date de modification):
+
+`ls -la`
+
+- Affiche les fichiers avec un format détaillé. Les tailles utilisent des prefixes d'unités pour améliorer la lisibilité (KB, MB, GB):
```

Comment:
Typo `utilisent` → `utilisant`. Also, maybe it sounds simpler like this?

```suggestion
- Affiche les fichiers avec un format détaillé utilisant préfixes d'unité pour la taille (KB, MB, GB):
```

Reply:
My French is limited, but that doesn't look like a typo to me 😄. AFAIK "(They) use" --> "ils [utilisent](https://en.wiktionary.org/wiki/utilisent)".

Other than that, I agree with the suggested change.

Reply:
@waldyrious you're right, got confused by my own rewording :')

Reply:
[Muphry's law](https://en.wikipedia.org/wiki/Muphry%27s_law) in action ;)

Reply:
Agree with a slight modification without the comma, it makes more sense to me.

Reply:
Nice, thank you :+1: